.pylintrc

```diff
@@ -354,7 +354,7 @@ max-attributes=16
 max-bool-expr=5
 
 # Maximum number of branch for function / method body
-max-branches=12
+max-branches=13
```

Comment:
incremented this so that `pretty_format()` can check the `maintainer` property

Reply:
please make it as inline comment in pretty_format

Reply:
ngl, i didn't even know you could do that. fixed in e56599f